quiz.js

```diff
@@ -50,7 +66,7 @@ db.dropDatabase();
 // your code begins here
 
 
-
+db.houses.update({ name: 'Arryn' }, { $unset: { motto : "As High as Honor" }});
```

Comment:
Technically, it's not necessary to list "As High As Honor" in this line; [$unset](http://docs.mongodb.org/manual/reference/operator/update/unset/) doesn't actually use that value at all. This will still work, though.